specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json

```diff
@@ -3761,7 +3761,8 @@
             "ModifyRequestHeader",
             "ModifyResponseHeader",
             "UrlRedirect",
-            "UrlRewrite"
+            "UrlRewrite",
+            "OriginGroupOverride"
```

Comment:
Adding an enum value to a discriminator will introduce a breaking change as the SDK won't know how to serialize the new discriminator type.

Please see 'New property added to response':
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
We have not released this API version in prod as yet. And no SDK's have been generated for this version of the API as yet publicly.